### DISCUSSION TITLE
Custom Close Button

### DISCUSSION THREAD

**Discussion by alberthenderson** [fancyapps/ui#32] [Q&A] [answered]
Previously on fancyapps v.3 active fancybox can be triggered to close with `parent.$.fancybox.close()`, now it's not working, is there any changes were made and probably I didn't notice it on documentation of fancyapps v.4?

**Comment by fancyapps** [comment, marked as answer]
Hi,

So, you want to close Fancybox from within the iframe? Then simply use `parent` with API any method (you can find list of methods here - https://fancyapps.com/docs/ui/fancybox/api), for example: `parent.Fancybox.close()`.

Keep in mind that Fancybox object will not be globally available by default if you are using ES6 modules. In that case, simply make it global - `window.Fancybox = Fancybox` - right after importing Fancybox in your main frame.

**Reply by alberthenderson** [reply]
Thank you for pointing it out. it work!!!
